src/physics/autompo.jl

```diff
@@ -131,26 +150,48 @@ Base.copy(ampo::AutoMPO) = AutoMPO(copy(data(ampo)))
 
 Base.size(ampo::AutoMPO) = size(data(ampo))
 
-function add!(ampo::AutoMPO,
-              op::String, i::Int)
-  push!(data(ampo),MPOTerm(1.0,op,i))
-  return
-end
 
+"""
+    add!(ampo::AutoMPO,
+         op::String, i::Int)
+
+    add!(ampo::AutoMPO,
+         coef::Number,
+         op1::String, i::Int)
+
+Add a single-site operator term
+to the AutoMPO `ampo`. The operator's
+name is `op` and site number is `i`.
+The second version accepts a real
+or complex coefficient.
+"""              
 function add!(ampo::AutoMPO,
               coef::Number,
               op::String, i::Int)
   push!(data(ampo),MPOTerm(coef,op,i))
   return
 end
 
-function add!(ampo::AutoMPO,
-              op1::String, i1::Int,
-              op2::String, i2::Int)
-  push!(data(ampo),MPOTerm(1.0,op1,i1,op2,i2))
-  return
-end
+add!(ampo::AutoMPO,op::String, i::Int) = add!(ampo,1.0,op,i)
+
+
+"""
+    add!(ampo::AutoMPO,
+         op1::String, i1::Int,
+         op2::String, i2::Int,
+         ops...)
 
+    add!(ampo::AutoMPO,
```

Comment:
Maybe it would be nice to mention the alternative syntax `.+=` here as well?

Reply:
Yes, I was actually wondering where to mention that. I’m not sure if people ever do doc strings for operators like ?+ so then the named version (such as add!) is a great place to mention it.

Reply:
Do you have a good suggestion for how to document it though? Like we could put
```
+(ampo::AutoMPO, term::Tuple)
```
there as well but it’s a bit cryptic. Maybe that plus a doc example later on will help.

Reply:
I think it is fine to put what you wrote, and then also doc examples.